### PR TITLE
Re welders

### PR DESCRIPTION
**Pull request by FluidRock**
## About the PR 

Added the ability to reverse engineer advanced welders along with a minor buff to the Experminetal Welding tool.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Reverse engineering machine can now analyze Industrial and Adavanced Industrial welding tool.

### CODE REVIEW

**Comment by Elijahrane** [comment]
image

This can be fixed by either just giving ifrit welder an RE of itself or removing its advanced welder parenting

**Comment by Vordenburg** [comment]
The infinite fuel welder probably shouldn't be reverse-engineerable. The ifrit hand could use its own entity at some point, but if the infinite fuel welder isn't made RE-able, then that becomes a much lower priority issue.

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml [hunk 1]
```diff
@@ -106,6 +110,11 @@
         maxVol: 250
   - type: Tool
     speed: 1.3
+  - type: ReverseEngineering
+    difficulty: 3
+    recipes:
+      - WelderIndustrialAdvanced
+
```
Comment (Vordenburg) [inline Resources/Prototypes/Entities/Objects/Tools/welders.yml:117]:
```suggestion
```

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml [hunk 2]
```diff
@@ -134,6 +143,8 @@
       reagents:
       - ReagentId: WeldingFuel
         Quantity: 1
+  - type: Tool
+    speed: 1.3
```
Comment (Vordenburg) [inline Resources/Prototypes/Entities/Objects/Tools/welders.yml]: This change should be directed to Wizard's Den.

**Review by Vordenburg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**